content/en/data_security/agent.md

```diff
@@ -68,25 +68,25 @@ Agent v6/7 comes bundled with a Graphical User Interface (GUI) by default, which
+Regarding its Container Agent specifically, Datadog performs regular vulnerability static analysis on both its general availability (GA) and release candidate (RC) releases. The Datadog Container Agent can be found in public registries as mentioned in [Docker Agent][10], and additionally, Datadog agent source code is open source. This empowers customers to perform vulnerability scanning with their preferred tooling based on a cadence that meets their unique needs. This provides the required visibility for customers inclined to monitor the Datadog agent for potential vulnerabilities.
 
-If you believe you've discovered a bug in Datadog's security, get in touch at [security@datadoghq.com][14] and we will get back to you within 24 hours. Datadog's [PGP key][15] is available for download in case you need to encrypt communications with us. We request that you not publicly disclose the issue until we have had a chance to address it.
+If you believe you've discovered a bug in Datadog's security, see [Report An Issue][11]. To submit a vulnerability inquiry on a specific product as an existing customer, reach out to [Datadog Support][12] through your standard support process. If submitting a support ticket through the Datadog website, set the `Product type` field to `Vulnerability Inquiry on Datadog Product`.
 
 ## Running as an unprivileged user
 
-By default, the Agent runs as the `dd-agent` user on Linux and as the `ddagentuser` account on [Windows][16]. The exceptions are as follows:
+By default, the Agent runs as the `dd-agent` user on Linux and as the `ddagentuser` account on [Windows][13]. The exceptions are as follows:
 
 - The `system-probe` runs as `root` on Linux and as `LOCAL_SYSTEM` on Windows.
 - The `process-agent` runs as `LOCAL_SYSTEM` on Windows.
 - The `security-agent` runs as `root` on Linux.
 
 ## Secrets management
 
-If you have a requirement to avoid storing secrets in plaintext in the Agent's configuration files, you can leverage the [secrets management][17] package. This package allows the Agent to call a user-provided executable to handle retrieval or decryption of secrets, which are then loaded in memory by the Agent. You can design your executable according to your preferred key management service, authentication method, and continuous integration workflow.
+If you have a requirement to avoid storing secrets in plaintext in the Agent's configuration files, you can leverage the [secrets management][14] package. This package allows the Agent to call a user-provided executable to handle retrieval or decryption of secrets, which are then loaded in memory by the Agent. You can design your executable according to your preferred key management service, authentication method, and continuous integration workflow.
 
-For more information, see the [Secrets Management][18] documentation.
+For more information, see the [Secrets Management][15] documentation.
```

Comment:
```suggestion
For more information, see the [Secrets Management][14] documentation.
```

Reply:
These two links both go to the same place, so I figured it made sense to remove the duplicate.